packages/breadcrumbs/src/styled/StyledBreadcrumb.ts

```diff
@@ -6,19 +6,22 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
-import { retrieveComponentStyles, isRtl } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.list';
 
-export const StyledBreadcrumb = styled.ol.attrs(props => ({
+/**
+ * 1. <ol> reset.
+ */
+export const StyledBreadcrumb = styled.ol.attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(BreadcrumbStyles['c-breadcrumb'], {
-    // RTL
-    [BreadcrumbStyles['is-rtl']]: isRtl(props)
-  })
-}))`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+  'data-garden-version': PACKAGE_VERSION
+})`
+  ${props => `
+    display: flex; /* [1] */
+    margin: 0; /* [1] */
+    padding: 0; /* [1] */
+    list-style: none; /* [1] */
+    font-size: ${props.theme.fontSizes.md};
+  `};
+  ${props => props.theme.rtl && 'direction: rtl'}
```

Comment:
This syntax you're using is a little different than what we do in the other components. I would use:

```tsx
styled.ol`
  display: flex;
  margin: 0;
  /* etc */
  font-size: ${props => props.theme.fontSizes.md};
  direction: ${props => props.theme.rtl && 'rtl'}
`;
```

if you have a block which uses _a lot_ of themed values you can use an alternative signature that is similar to

```tsx
styled.ol(props => `
  display: flex;
  font-size: ${props.theme.fontSizes.md};
  /* etc */
`);
```

Reply:
:+1: to @austin94's comments here. Please remain consistent with other refactored CSS-in-JS components.

Reply:
Thanks for the suggestion @austin94 — I'll make sure to be consistent with the existing code.